Dockerfile

```diff
@@ -3,7 +3,7 @@ ADD . /build
 WORKDIR /build
 RUN make build
 
-FROM gcr.io/distroless/base:debug AS trento-web
+FROM gcr.io/distroless/base:debug AS telemetry-collector
 COPY --from=go-build /build/telemetry /app/telemetry
```

Comment:
I think this was a leftover of my copy paste from the trento Dockerfile, we could also skip the `AS` directive or I would use `telemetry` as the target name

Reply:
changed to `telemetry`